scheduler/scheduler-server/src/test/java/functionaltests/scripts/TestTaskScriptVariables.java

```diff
@@ -87,10 +87,13 @@ public void testTaskVariables() throws Throwable {
         String[] logsLines = logs.split(System.lineSeparator());
         TaskLogs outputVariables = schedulerHelper.getTaskResult(id, "taskVariables").getOutput();
         String[] outputVariablesLines = outputVariables.getStdoutLogs(false).split(System.lineSeparator());
+        TaskLogs outputChild = schedulerHelper.getTaskResult(id, "childTask").getOutput();
```

Comment:
Additionally, please change the header class explanation to describe what the test is doing.